src/core/include/openvino/runtime/tensor.hpp

```diff
@@ -137,6 +154,13 @@ class OPENVINO_API Tensor {
      */
     Shape get_shape() const;
 
+    /**
+     * @brief Copy tensor
+     *
+     * @param dst destination tensor, should have the same element type and shape
+     */
```

Comment:
```suggestion
    /**
     * @brief Copy tensor's data into another tensor. Destination tensor should have the same element type and shape.
     *
     * @param dst destination tensor.
     */
```
I think comment should be more descriptive about action.

What about strides if `copy_to` is used?

Reply:
Strides can be different. Also I will create a ticket to extend tensor API in python